### ISSUE TITLE
Feature/ Duplicate animation

### ISSUE THREAD

**Issue by vanlooverenkoen** [alexjlockwood/ShapeShifter#231]
I am working on an animation with almost always the same values, now I have to write everyting a 1000 times. It would be cool to see a dublicate action on every animation which copies all the values and pasts it after the original one if there is still time to execute the duplicate.

**Comment by alexjlockwood** [comment]
Do you mean the ability to copy and paste the draggable animation blocks in the bottom timeline panel?

**Comment by vanlooverenkoen** [comment]
Yes that is what I mean. Copy and pasting could be nice as well. Maybe selecting 10 blocks copy past and you got a duplicate of everything. 

It would also be nice to be able to select multiple your animation blocks of different paths and copy and past them somewhere else.

**Comment by alexjlockwood** [comment]
Yup, definitely something I want to add as well. I'm currently rewriting a significant portion of the web app, so hopefully I can squeeze this in as I'm doing it!
